cytominer_eval/tests/test_evaluate.py

```diff
@@ -215,3 +215,75 @@ def test_evaluate_grit():
             grit_control_perts=grit_compound_control_perts,
         )
     assert "For grit, replicate_groups must be a dict" in str(ae.value)
+
+
+def test_evaluate_mp_value():
+    # Permutations in mp_value could lead to some edge cases
+    np.random.seed(2020)
+    
+    # Tests on genetic screen dataset
+    mp_value_gene_control_perts = [
+        "Chr2-1",
+        "Chr2-2",
+        "Chr2-3",
+        "Chr2-4",
+        "Chr2-5",
+        "Chr2-6",
+        "Luc-1",
+        "Luc-2",
+        "LacZ-2",
+        "LacZ-3",
+    ]
+    
+    mp_value_gene_replicate_groups = "Metadata_pert_name"
+
+    mp_value_results_df = evaluate(
+        profiles=gene_profiles,
+        features=gene_features,
+        meta_features=gene_meta_features,
+        replicate_groups=mp_value_gene_replicate_groups,
+        operation="mp_value",
+        grit_control_perts=mp_value_gene_control_perts,
+    )
+    
+    # Negative controls should be "close to themselves"
+    assert all(
+        mp_value_results_df[[
+            x in mp_value_gene_control_perts for 
+            x in mp_value_results_df.Metadata_pert_name
+                           ]].mp_value >= 0.05)
+    
+    # Strong perturbation should differ from controls
+    assert "PTK2-2" in list(mp_value_results_df[
```

Comment:
wait, I thought that a low mp_value meant an inactive perturbation. Is this a wrong interpretation?